languages/csharp/exercises/concept/interfaces/.docs/after.md

```diff
@@ -0,0 +1,170 @@
+`interfaces` are the primary means of [decoupling][wiki-loose-coupling] the uses of a class from its implementation. This decoupling provides flexibility for maintenance of the implementation and helps support type safe generic behavior.
```

Comment:
Is there perhaps a document to link to regarding interfaces in C#?

Reply:
Done